code/parachain/frame/pablo/src/test/dual_asset_constant_product_tests.rs

```diff
@@ -322,55 +307,197 @@ fn test_add_liquidity_with_disproportionate_amount() {
 #[test]
 fn add_lp_with_min_mint_amount() {
 	new_test_ext().execute_with(|| {
-		let pool_init_config = valid_pool_init_config(
-			&ALICE,
-			BTC,
-			Permill::from_percent(50_u32),
-			USDT,
-			Permill::zero(),
+		next_block::<Pablo, Test>();
+
+		let first_asset = BTC;
+		let second_asset = USDT;
+
+		let pool_init_config = PoolInitConfiguration::DualAssetConstantProduct {
+			owner: ALICE,
+			assets_weights: dual_asset_pool_weights(
+				first_asset,
+				Permill::from_percent(50_u32),
+				second_asset,
+			),
+			fee: Permill::zero(),
```

Comment:
Perhaps we should collect all testing related comments into a ClickUp mega-tasks for me and @benluelo to pick up so that QA and others can be unblocked while we refine and update the tests suite.

Reply:
Ok with however the way you guys want to handle